apps/drupal-default/particle_theme/templates/paragraphs/image-hover/block--utc-hover-images.html.twig

```diff
@@ -0,0 +1,36 @@
+{% extends directory ~ "/templates/block/block.html.twig" %}
```

Comment:
I never seen directory or ~. 
I think you could do just `{% extends "block.html.twig" %}`
That may be causing some issue although its working somewhat.

Reply:
Because we moved all the twig files for this component into one folder, the block twig is no longer in the block folder where that extended file resides. This is the code that tells Drupal where to find that extended file. I have to have this extension in order to edit it in Layout Builder.